src/components/LeaderBoard/LeaderBoard.container.jsx

```diff
@@ -23,7 +23,7 @@ const mapStateToProps = (state) => {
       element.intangibletimewidth = _.round((element.totalintangibletime_hrs * 100) / maxTotal, 0);
 
       element.barcolor = getcolor(element.totaltangibletime_hrs);
-      element.barprogress = getprogress(element.totaltangibletime_hrs);
+      element.barprogress = getProgressValue(element.totaltangibletime_hrs,40);
```

Comment:
Nice updates! Please also pay attention to the formatting details, like there is a space needed before 40. I believe Prettier can help you fix those issues.
<img width="757" alt="image" src="https://user-images.githubusercontent.com/5071040/210450084-c58f232d-359d-4343-aff2-45ca33f4d603.png">